plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/core/ResetBeanPostProcessorCaches.java

```diff
@@ -66,10 +67,10 @@ public static void reset(DefaultListableBeanFactory beanFactory) {
                     declaredMethodsCache.setAccessible(true);
```

Comment:
pls. use 'ReflectionHelper.getNoException() to fetch 'declaredMethodsCache' too.

Reply:
done